scripts/odf_sbe_metadata.py

```diff
@@ -29,19 +30,20 @@
 PKL_EXT = 'pkl'
 
 
-
+# these two can be removed
```

Comment:
debugPrint gets called a bunch of times in convert.py

Reply:
updated to be "...removed in favor of logger"